client/src/pages/TaggingPage.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from "react";
 import { useEffect } from "react";
```

Comment:
```suggestion
import React, { useState, use Effect } from "react";
```

you can import these in 1 line :)